var/spack/repos/builtin/packages/py-altair/package.py

```diff
@@ -9,17 +9,36 @@
 class PyAltair(PythonPackage):
     """Declarative statistical visualization library for Python"""
 
-    pypi = "altair/altair-4.2.0.tar.gz"
+    pypi = "altair/altair-5.2.0.tar.gz"
 
     license("BSD-3-Clause")
 
+    version("5.2.0", sha256="2ad7f0c8010ebbc46319cc30febfb8e59ccf84969a201541c207bc3a4fa6cf81")
+    version("5.1.2", sha256="e5f52a71853a607c61ce93ad4a414b3d486cd0d46ac597a24ae8bd1ac99dd460")
+    version("5.1.1", sha256="ad6cd6983c8db69a34dd68e42653f6172b7fc3775b7190005107f1b4fc60d64d")
+    version("5.1.0", sha256="46d2b1a9fa29eeed24513262cb1de13a40d55c04580fc21799d5de3991fea8ff")
+    version("5.0.1", sha256="087d7033cb2d6c228493a053e12613058a5d47faf6a36aea3ff60305fd8b4cb0")
+    version("5.0.0rc3", sha256="93ff660e45ad5b783057d331ddc489f9331cdf121e8450ddba2b71b35242d708")
```

Comment:
We don't normally add release candidates, especially when the actual release already exists. Do you need these for some reason?

Reply:
no, it is automatically generated by spack checksum, and I will remove it